Misc/NEWS.d/next/Tests/2023-07-13-02-14-57.gh-issue-99242.DHVhbD.rst

```diff
@@ -0,0 +1 @@
+When running regression tests under certain conditions (such as chroot) it was possible that calls to `os.getloadavg` would throw. This PR catches OS errors as reporting load average is optional.
```

Comment:
```suggestion
:func:`os.getloadavg` may throw :exc:`OSError` when running regression tests under
certain conditions (e.g. chroot). This error is now caught and ignored, since
reporting load average is optional.
```
How about this? We usually prefer a single line to be no longer than 79 characters.